library/task-library/K8 Module Escripts/Create_namespace.py

```diff
@@ -0,0 +1,27 @@
+#script
+
+#Variables used in this script 
+KUBERNETES_CLUSTER_IP=""
+API_KEY=""
+NAMESPACE_NAME=""
+
+
+import kubernetes.client
+from kubernetes.client.rest import ApiException
```

Comment:
these imports won't work in escript

Reply:
@balugeorge Recently team added support, please refer https://jira.nutanix.com/browse/CALM-16354